stories/CharacterSheet.stories.tsx

```diff
@@ -178,6 +178,15 @@ export const TheWitchIsDead = Template.bind({});
     return { character };
   },
 ];
+export const EdgeOfTheEmpire = Template.bind({});
+(EdgeOfTheEmpire as any).loaders = [
+  async () => {
+    const character = await CharacterFactory.make(
+      CharacterTemplates.EdgeOfTheEmpire
+    );
+    return { character };
+  },
+];
```

Comment:
**question** did you test that the template loaded correctly inside storybook by running `yarn storybook` ?